SPCTR-3430: Fix - Image Gallery custom URL validation issue (#3545)
brainstormforce/ultimate-addons-for-gutenberg: README.md

### x.x.x ###
* Improvement: Buttons - Added a new vertical alignment option.

> * Fix: Image Gallery - Resolved an issue in which the customLink validation is failing when the root domain has hyphen in it.

Reviewer: ```suggestion
* Fix: Image Gallery - Resolved an issue in which the custom link validation would fail when the root domain contained hyphens.
```

Author: Done.
